app/Providers/AppServiceProvider.php

```diff
@@ -16,5 +18,13 @@ public function register(): void
     public function boot(): void
     {
         JsonResource::withoutWrapping();
+        VerifyEmail::toMailUsing(function (object $notifiable, string $url) {
```

Comment:
Refactor the callback to use a shorter arrow function (fn) with a return type hint. This makes the code more concise and ensures type safety by explicitly returning a MailMessage object